test/helpers/common.js

```diff
@@ -776,3 +776,27 @@ export function triggerTouchEvent(type, target, pageX, pageY) {
 export function createSpreadsheetData(...args) {
   return Handsontable.helper.createSpreadsheetData(...args);
 }
+
+export class DataTransferObject {
```

Comment:
I think that we don't need to export this class.
```suggestion
class DataTransferObject {
```